_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,365 @@
+  visualizations. It is written in Rust and leverages the
+  rust-miniscript library and the open source Esplora block explorer.
+  ([transcript][mb ts], [video][mb vid])
+
+- **Watchtowers and BOLT13:** Sergi Delgado appeared on Potzblitz to
+  discuss the latest state of [watchtower][topic watchtowers]
+  development and a proposed watchtower protocol specification. He
+  explored the various trade-offs when designing a watchtower and the
+  interplay between privacy requirements, accessibility, storage, and
+  fees charged.  Delgado is working on the watchtower implementation
+  [The Eye of Satoshi][teos] at Talaia Labs, which is aiming to be
+  compliant with the proposed BOLT13 (see [Newsletter #75][news75
+  watchtower bolt]).  Delgado also highlighted how watchtowers are
+  becoming increasingly critical in multiple settings such as Bitcoin
+  [vault][topic vaults] designs, statechains, and atomic swaps in addition to LN.
+  ([transcript][watchtowers ts], [video][watchtowers vid],
+  [slides][watchtowers slides])
+
+- **CoinSwap:** Aviv Milner presented on CoinSwap at the Wasabi Research
+  Club. He explained the property of covertness and how Chris Belcher’s
+  CoinSwap proposal (see [Newsletter #100][news100 coinswap]) provides
+  covertness in a manner that other privacy schemes such as
+  [coinjoin][topic coinjoin] fail to do. Milner also went through the
+  motivation for routing CoinSwaps, namely to address the concern of
+  unwittingly entering into a CoinSwap with an adversary such a chain
+  surveillance company.  ([transcript][coinswap ts], [video][coinswap
+  vid])
+
+- **Taproot and schnorr multisignatures:** Some of the co-authors of [BIP340][],
+  including Tim Ruffing, Pieter Wuille, and Jonas Nick, held a wide ranging
+  discussion at London Bitcoin Devs on the specification of [schnorr
+  signatures][topic schnorr signatures].  This covered some of the
+  earlier ideas for implementing schnorr in Bitcoin and what the
+  co-authors thought the community should be concerned about when
+  considering a possible future soft fork deployment. Wuille expressed
+  his view that it is unlikely at this stage that there are bugs in the
+  proposed consensus implementation and that he is more concerned with
+  how schnorr is adopted and what is built using it.
+  ([transcript][socratic ts], [video][socratic vid])
+
+    The following day, Tim Ruffing presented on multisignature and
+    threshold signature schemes using schnorr signatures. He highlighted
+    how the rogue key attack and an attack using Wagner’s algorithm (see
+    [Newsletter #74][news74 wagner]) need to be addressed when designing
+    secure multisignature schemes.  Ruffing has been working with
+    collaborators on a speculative [MuSig][topic musig] signature scheme
+    that only requires two rounds of interaction rather than three
+    (without relying on zero knowledge proofs). Threshold signature
```

Comment:
This shouldn't be in a parenthetical. Better would be "..rather than three, without relying..."